tor/src/main/java/bisq/tor/Constants.java

```diff
@@ -56,7 +56,7 @@
     public final static String TORRC_KEY_COOKIE = "CookieAuthFile";
 
     // Tor control connection
-    public final static String CONTROL_PORT_LOG_SUB_STRING = "Control collection listening on port ";
+    public final static String CONTROL_PORT_LOG_SUB_STRING = "Control listener listening on port ";
```

Comment:
Oh that was a mistake happened when refactoring....

Reply:
Ok that explains it :+1: